### PR TITLE
UI rework

### PR DESCRIPTION
**Pull request by comzeradd**
- Update front page
- Use templates inheritance
- Add events form to choose events and voting method

### CODE REVIEW

**Comment by comzeradd** [comment]
@johngian r?

File: woodstock/voting/views.py [hunk 1]
```diff
@@ -33,6 +36,16 @@ def _get_percentage(partial, total):
 
 
 @login_required
+def events(request):
+    user = request.user
+    events = Event.objects.all()
+
+    return render(request, 'events.html',
+                  {'user': user,
```
Comment (johngian) [inline woodstock/voting/views.py]: Please use a single line for this dict.

File: woodstock/voting/views.py [hunk 2]
```diff
@@ -1,11 +1,14 @@
 from django.contrib import messages
 from django.contrib.auth.decorators import login_required
 from django.shortcuts import redirect, render, get_object_or_404
+from django.views.decorators.http import require_http_methods
```
Comment (johngian) [inline woodstock/voting/views.py]: Unused dependencies.

Comment (johngian) [inline woodstock/voting/views.py]:
```
woodstock/voting/views.py:4:1: F401 'require_http_methods' imported but unused
woodstock/voting/views.py:5:1: F811 redefinition of unused 'messages' from line 1
```

**Comment by johngian** [comment]
Logout functionality broken (probably because of the js libs definition).

File: woodstock/voting/static/js/events.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        $('input:checkbox:checked').each(function(){
+            events.push($(this).val());
+        });
+        var blind = $('input:radio:checked').val();
+
+        $('#events-submit').attr('href', '/dashboard/?events=' + events + '&blind=' + blind);
+    }
+
+    $('input').change(function() {
+        generate_click();
```
Comment (johngian) [inline woodstock/voting/static/js/events.js]: No need to define such a specific method. Move `generate_click` definition here.

**Comment by comzeradd** [comment]
@johngian PR updated. Logout fixed.

**Comment by johngian** [comment]
@comzeradd 
Code looks good for now.
r+

**Pull request merged**